qiskit/visualization/pulse_v2/core.py

```diff
@@ -243,6 +243,22 @@ def _waveform_loader(self, program: Union[pulse.Waveform, pulse.ParametricPulse]
 
         self.charts.append(chart)
 
+    @staticmethod
+    def _flatten_frames(program: pulse.Schedule) -> pulse.Schedule:
```

Comment:
Why we cannot call `pulse.transforms` here? Seems like we have the same functionality in two places.